src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -55,15 +55,15 @@ import styles from './Browser.scss';
 import 'static/browser/browser.js';
 
 type StateProps = {
-  activeGenomeId: string | null;
+  activeGenomeId: string;
   activeEnsObjectId: string | null;
   allActiveEnsObjectIds: { [genomeId: string]: string };
   browserActivated: boolean;
   browserNavOpened: boolean;
   browserQueryParams: { [key: string]: string };
   chrLocation: ChrLocation | null;
   allChrLocations: { [genomeId: string]: ChrLocation };
-  drawerOpened: boolean;
+  drawerOpened: { [genomeId: string]: boolean };
```

Comment:
Why are we getting the whole object of all drawer states for all genomeId? Aren’t we interested only in one, for the active genome?

I vote for having it as a boolean. Also, if possible, can we have it with an auxiliary verb (`isDrawerOpened`) so that it reads as a question?